lib/phoenix_live_view/html_engine.ex

```diff
@@ -82,21 +82,22 @@ defmodule Phoenix.LiveView.HTMLEngine do
   @doc false
   def handle_end(state) do
     state
+    |> Map.put(:root, false)
```

Comment:
Please pass it as non-optional parameter on token_state instead? :)

Reply:
Sure!